doc/make.py

```diff
@@ -25,7 +25,7 @@ def latex():
 
         os.chdir('../..')
     else:
-        print 'latex build has not been tested on windows'
+        print('latex build has not been tested on windows')
```

Comment:
Need to add a `from __future__ import print_function` at the top so that this will still work in the python 2 series.

Reply:
done